stubs/click/click/testing.pyi

```diff
@@ -53,7 +54,7 @@ class CliRunner:
     def make_env(self, overrides: Optional[Mapping[str, str]] = ...) -> Dict[str, str]: ...
     def isolation(
         self, input: Optional[Union[bytes, Text, IO[Any]]] = ..., env: Optional[Mapping[str, str]] = ..., color: bool = ...
-    ) -> ContextManager[BinaryIO]: ...
+    ) -> ContextManager[Tuple[io.BytesIO, io.BytesIO]]: ...
```

Comment:
The second element of the tuple could also be False, right? It might work out better in practice to lie a little and claim it's also a BytesIO, but in general we prefer not to lie in typeshed.

Reply:
I went ahead and changed it to a Union including False to unblock this PR.